app/services/bing/search_service.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Bing
+  class SearchService
+    USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) '\
+                 'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36'
+    BASE_SEARCH_URL = 'https://www.bing.com/search'
+    HEADERS = { 'User-Agent' => USER_AGENT }.freeze
+
+    def initialize(keyword)
```

Comment:
It may confuse the developer, should we pass a keyword instance to this param or keyword name?

```suggestion
    def initialize(keyword_name)
```

As far as i check above, we passed keyword name, not the keyword instance.
But personally, i would prefer to pass a `keyword instance` to this service initialization, because maybe we could use some additional attribute on the `keyword instance`

Reply:
solved in 7e6b0a1